lib/realtime/api.ex

```diff
@@ -202,4 +216,12 @@ defmodule Realtime.Api do
     |> Task.await_many()
     |> List.flatten()
   end
+
+  def drop_dist_cache(id) do
+    {_, bad_nodes} =
+      [node() | Node.list()]
+      |> :rpc.multicall(Cachex, :del, [:db_cache, id], 10_000)
+
+    if bad_nodes != [], do: Logger.error("Failed to drop cache on nodes: #{inspect(bad_nodes)}")
```

Comment:
@abc3 `bad_nodes` are a list of nodes that do not exist. It's equally, or more, important to log the badrpc reasons on nodes this call was able to reach.

Reply:
`bad_nodes` is a list of cluster members which didn't answer

Reply:
```
It returns {ResL, BadNodes}, where BadNodes is a list of the nodes that do not exist, and ResL is a list of the return values, or {badrpc, Reason} for failing calls.
```
- https://www.erlang.org/doc/man/rpc.html#multicall-5

Let's log both.

Reply:
You can reproduce a bad node if you would try to exec a multicall function on timer.sleep which is bigger than the timeout
```erlang
:rpc.multicall(Node.list(), :timer, :sleep, [1000], 500)
{[], [:"node@127.0.0.1"]}
```
ReaL does not include a node name
```erlang
:rpc.multicall(Node.list(), Map, :new, [1], 500)
{[
   badrpc: {:EXIT,
    {%Protocol.UndefinedError{protocol: Enumerable, value: 1, description: ""},
     [
       {Enumerable, :impl_for!, 1, [file: 'lib/enum.ex', line: 1]},
       {Enumerable, :reduce, 3, [file: 'lib/enum.ex', line: 166]},
       {Enum, :reverse, 1, [file: 'lib/enum.ex', line: 4307]},
       {Enum, :to_list, 1, [file: 'lib/enum.ex', line: 3646]},
       {Map, :new_from_enum, 1, [file: 'lib/map.ex', line: 228]},
       {Map, :new, 1, []}
     ]}}
 ], []}
```